src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -866,7 +866,7 @@ private static void ValidateFileFrameworks(IEnumerable<IPackageFile> files)
         /// <exception cref="PackagingException">When a validation rule is not met</exception>
         private void ValidateReadmeFile(IEnumerable<IPackageFile> files, string readmePath)
         {
-            if (!string.IsNullOrEmpty(readmePath))
+            if (!PackageTypes.Contains(PackageType.SymbolsPackage) && !string.IsNullOrEmpty(readmePath))
```

Comment:
The summary comment block should say something about this; eg, "Validate readme....except for Symbols packages."

Reply:
Thanks! Added.